src/openforms/formio/utils.py

```diff
@@ -241,6 +241,8 @@ def is_visible_in_frontend(component: JSONObject, data: DataMapping) -> bool:
             "eq": <compare value>
         }
 
+    .. warning:: this function currently does not take parent components into account
```

Comment:
Isn't this the whole point of the ticket?

Reply:
the [method on the `FormioConfigurationWrapper`](https://github.com/open-formulieren/open-forms/pull/2408/files#diff-3082701c8fd1dee09c8388c9d4de636a24ee32cacec78822a0aeecde0f9969feR71) _does_ take it into account and it uses this function. The test proves it works with parent components now :)